src/main/java/org/apache/sysml/parser/Statement.java

```diff
@@ -61,7 +61,39 @@
 	public static final String GAGG_FN_CM       = "centralmoment";
 	public static final String GAGG_FN_CM_ORDER = "order";
 	public static final String GAGG_NUM_GROUPS  = "ngroups";
-	
+
+	// String constants related to parameter server builtin function
+	public static final String PS_MODEL = "model";
+	public static final String PS_FEATURES = "features";
+	public static final String PS_LABELS = "labels";
+	public static final String PS_VAL_FEATURES = "val_features";
+	public static final String PS_VAL_LABELS = "val_labels";
+	public static final String PS_UPDATE_FUN = "upd";
+	public static final String PS_AGGREGATION_FUN = "agg";
+	public static final String PS_MODE = "mode";
+	public static final String PS_MODE_LOCAL = "LOCAL";
+	public static final String PS_MODE_REMOTE_SPARK = "REMOTE_SPARK";
```

Comment:
You might want to define an enum such as PSModeType because we will use that many times.